lib/alex_koin/slack.ex

```diff
@@ -60,21 +79,35 @@ defmodule AlexKoin.SlackRtm do
 
     # validate this coin belongs to the user currently
     if coin.wallet_id == user_wallet(user).id do
-      SlackCommands.transfer_coin(coin, user_wallet(user), user_wallet(to_user), captures["memo"])
+      memo = captures["memo"]
+      SlackCommands.transfer_coin(coin, user_wallet(user), user_wallet(to_user), memo)
+
+      # Notify the recipient of the new koin
```

Comment:
Don't leave commented code in. Pure cruft, or it belongs over in a branch if it's a feature in development.